Src/Sarif.PatternMatcher/ValidatorsCache.cs

```diff
@@ -92,9 +92,16 @@ public ValidatorsCache(IEnumerable<string> validatorBinaryPaths = null, IFileSys
                     Environment.CurrentDirectory = Path.GetDirectoryName(location);
                 }
 
-                validationState =
-                    (ValidationState)isValidStaticMethodInfo.Invoke(
-                        obj: null, arguments);
+                try
+                {
+                    validationState =
+                        (ValidationState)isValidStaticMethodInfo.Invoke(
+                            obj: null, arguments);
+                }
+                catch (TargetInvocationException e)
+                {
+                    throw e.InnerException;
```

Comment:
>InnerException [](start = 28, length = 14)

Now the console output and SARIF logs will show us the actual exception in the called component. Hooray!

Reply:
should we move this to the previous try/finally?

---
In reply to: [604541612](https://github.com/microsoft/sarif-pattern-matcher/pull/349#discussion_r604541612) [](ancestors = 604541612)

Reply:
Agree that a single try/catch is cleaner and I'm pretty sure behaviorally equivalent

---
In reply to: [604854469](https://github.com/microsoft/sarif-pattern-matcher/pull/349#discussion_r604854469) [](ancestors = 604854469,604541612)

Reply:
Yes! good note, will do.

---
In reply to: [605224616](https://github.com/microsoft/sarif-pattern-matcher/pull/349#discussion_r605224616) [](ancestors = 605224616,604854469,604541612)